.github/workflows/build-and-test.yaml

```diff
@@ -30,11 +30,14 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '14.17.0'
-      - name: build shared-components
+      - name: "Prepare local dependencies for shared packages"
+        run: node bin/prepare-production-build.mjs
```

Comment:
prepare-production-build.mjs er et nytt skript som gjør at vi slipper å kalle deploy-context.mjs for hver enkelt pakke